code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -223,14 +223,24 @@
 	var/distance = get_dist(user,src)
 	if(istype(user, /mob/abstract/observer) || user.stat == 2) // ghosts can see anything
 		distance = 1
-	if (src.stat && !(src.species.flags & NO_BLOOD))	// No point checking pulse of a species that doesn't have one.
+	var/mob/living/carbon/human/H = user
```

Comment:
Is this typechecked properly above?

Reply:
If you mean if the user is ``/mob/living/carbon/human/user`` by default, then no. Or at least I don't think it is. Unless I'm severely misunderstanding what you mean

Reply:
We need to check if the user is actually a human then, no? Otherwise this is going to runtime later on if a ghost does it.